vnext/Microsoft.ReactNative/JsiReader.cpp

```diff
@@ -72,19 +76,32 @@ bool JsiReader::GetNextArrayItem() noexcept {
   }
 
   auto &top = m_containers[m_containers.size() - 1];
-  if (top.Type != ContainerType::Array) {
+  if (top.Type == ContainerType::Object) {
```

Comment:
Should this also check for other primtive values like numbers and functions?

Reply:
maybe better to add a 'default' case in the switch to return false and move top.index++ inside the current two cases?

---
In reply to: [441651205](https://github.com/microsoft/react-native-windows/pull/5266#discussion_r441651205) [](ancestors = 441651205)

Reply:
Appologies, I only now read ContainerType, not NodeTYpe...

---
In reply to: [441651850](https://github.com/microsoft/react-native-windows/pull/5266#discussion_r441651850) [](ancestors = 441651850,441651205)